test/unittest/misc/test_dataclasses.py

```diff
@@ -3,6 +3,7 @@
 from pydantic import Field, dataclasses as dc
 
 
+# TODO: Remove this because it's not a relevant test
```

Comment:
That's fine -- you can remove anything you think is not relevant, and doesn't improve
the thorough-ness or coverage of the test-suite.